Sources/WasmKit/DebuggerInstructionMapping.swift

```diff
@@ -0,0 +1,86 @@
+
+/// Two-way mapping between Wasm and internal iseq bytecode instructions. The implementation of the mapping
+/// is private and is empty when `WasmDebuggingSupport` package trait is disabled.
+struct DebuggerInstructionMapping {
```

Comment:
Nice side-effect of moving to a separate type is that it's easier to switch to an empty implementation of the mapping based on package traits.

@kateinoigakukun I hope this addresses your review feedback, let me know if anything else is needed to move this forward.